src/game/actions.ts

```diff
@@ -15,4 +15,9 @@ export interface MagicAction {
   spell: Spell;
 }
 
-export type Action = StartNewGameAction | MoveAction | MagicAction;
+export interface CheatAction {
```

Comment:
Nobody likes a cheater, but in the 80s cheaters were kings

Reply:
I don't think this has changed in the last 3 decades. 

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/70121974-e0cd-45a5-900a-686a12c74905/thread/864d7b72-c3ad-4e32-89c4-01260321e1db?message=a218f3dc-a172-4899-8840-69e72193a453).

Reply:
Nooo!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/3f589ab2-072e-4d5b-abfb-dc72ea270639?message=85fb6e34-4d7f-4f28-95ad-e56287b1175b).